crypto/modes/asm/ghash-s390x.pl

```diff
@@ -132,10 +113,22 @@
 					#  vector
 	tmhh	%r0,0x4000	# check for function 65
 	jz	.Lsoft_ghash
+	# Parameter Block:
+	# Chaining Value (XI) 128byte
+	# Key (Htable[8]) 128byte
+	lmg	%r0,%r1,0($Xi)
+	stmg	%r0,%r1,8($sp)
+	lmg	%r0,%r1,8*16($Htbl)
+	stmg	%r0,%r1,24($sp)
+	la	%r1,8($sp)
 	lghi	%r0,S390X_GHASH	# function 65
-	la	%r1,0($Xi)	# H lies right after Xi in gcm128_context
+	# Do not assume this function is called from a gcm128_context.
+	# This is not true, e.g., for AES-GCM-SIV.
+	#la	%r1,0($Xi)	# H lies right after Xi in gcm128_context
```

Comment:
Generally we prefer to remove dead code. I'm not sure what benefit leaving this here adds.

Reply:
Yes, I will remove this.  Left in it for easier review and explanation of the change.

Reply:
Removed the code and moved the comment to the correct position.